### PR TITLE
Quota changes

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

https://transformuk.atlassian.net/browse/HOTT-712

### What?

I have added/removed/altered:

- [x] Updates quota order number view to match new content requirements
- [x] Adds helper methods for quota measures and specs
- [x] Adds new Time format that is gov compliant

Screenshot from 2021-05-26 14-29-40

### Why?

I am doing this because:

- This is a business requirement

### CODE REVIEW

File: app/helpers/measures_helper.rb [hunk 1]
```diff
@@ -1,12 +1,41 @@
 module MeasuresHelper
-  def order_number_info(record)
-    record.definition.try(:description) || "Order number #{record.number}"
+  def show_warning?(order_number)
+    order_number.definition.description && order_number.number.start_with?('05')
```
Comment (krios82) [inline app/helpers/measures_helper.rb:3]: Are we sure the `order_number.definition` and `order_number.number` will never be nil?

File: app/views/order_numbers/_order_number.html.erb [hunk 2]
```diff
@@ -1,63 +1,55 @@
...
+            <th scope="col" class='govuk-table__header'>Blocking period</th>
+            <td class="numerical govuk-table__cell numerical"><%= blocking_period_dates_for(order_number.definition) %></td>
           </tr>
         </tbody>
       </table>
-      <p class="govuk-!-margin-top-4">In an effort to facilitate UK importers HMRC endeavours to keep quotas 'OPEN' where possible, therefore the information displayed above may differ slightly from the current UK position. If you require further information contact the Tariff Quota Management Team by email: <%= mail_to("tariff.management@hmrc.gsi.gov.uk", nil, subject: "Order number #{order_number.number}") %>.</p>
+      <p class="govuk-!-margin-top-4">In an effort to facilitate UK importers HMRC endeavours to keep quotas 'OPEN' where possible, therefore the information displayed above may differ slightly from the current UK position. If you require further information contact the Tariff Quota Management Team by email: <%= mail_to("tariff.managment@hmrc.gov.uk", nil, subject: "Order number #{order_number.number}") %>.</p>
```
Comment (krios82) [inline app/views/order_numbers/_order_number.html.erb]: The `managment` part of this address seems misspelled.
I've seen it's also misspelled in the ticket, is the official address that is misspelled or it's a typo from Matt?

File: app/helpers/measures_helper.rb (hunk 1 above)
Comment (tavyy) [reply app/helpers/measures_helper.rb:3]: Yeah, good point I think we need to guard against those cases.

File: app/helpers/measures_helper.rb [hunk 3]
```diff
@@ -1,12 +1,41 @@
...
+
+  def suspension_period_dates_for(definition)
+    if definition.suspension_period_start_date.present? && definition.suspension_period_end_date.present?
+      start_date = definition.suspension_period_start_date
+      end_date = definition.suspension_period_end_date
+
+      "#{start_date.to_formatted_s(:gov)} to #{end_date.to_formatted_s(:gov)}"
+    else
+      'n/a'
+    end
```
Comment (tavyy) [inline app/helpers/measures_helper.rb:21]: Fancy using:

```ruby
 return 'n/a' unless foo

 bar
```
?

Not a blocker.

File: app/helpers/measures_helper.rb (hunk 1 above)
Comment (willfish) [reply app/helpers/measures_helper.rb:3]: No no, we do in the view already so are already being defensive

File: app/views/order_numbers/_order_number.html.erb (hunk 2 above)
Comment (willfish) [reply app/views/order_numbers/_order_number.html.erb]: I clarified with Matt and this should have the e in it. This is the spelling in the ticket

Comment (willfish) [reply app/views/order_numbers/_order_number.html.erb]:
```suggestion
      <p class="govuk-!-margin-top-4">In an effort to facilitate UK importers HMRC endeavours to keep quotas 'OPEN' where possible, therefore the information displayed above may differ slightly from the current UK position. If you require further information contact the Tariff Quota Management Team by email: <%= mail_to("tariff.management@hmrc.gov.uk", nil, subject: "Order number #{order_number.number}") %>.</p>
```

File: app/helpers/measures_helper.rb (hunk 3 above)
Comment (willfish) [reply app/helpers/measures_helper.rb:21]: I did originally and found it less readable :thinking:

Comment (willfish) [reply app/helpers/measures_helper.rb:21]: I think rust and elixir have ruined me

File: app/helpers/measures_helper.rb (hunk 1 above)
Comment (willfish) [reply app/helpers/measures_helper.rb:3]: It's not possible for the number to be nil

**Review by krios82 — APPROVED** [review]

**Pull request merged**